app/react/package.json

```diff
@@ -34,9 +34,9 @@
     "@svgr/webpack": "^4.0.3",
     "babel-plugin-named-asset-import": "^0.3.1",
     "babel-plugin-react-docgen": "^3.0.0",
-    "babel-preset-react-app": "^7.0.2",
+    "babel-preset-react-app": "^8.0.0",
     "common-tags": "^1.8.0",
-    "core-js": "^2.6.5",
+    "core-js": "^3.0.1",
```

Comment:
@Spy-Seth, it might be wise to pin this to the exact version used, as in CRA:
https://github.com/facebook/create-react-app/blob/265c1592dcf3853122dcdd50e1bfc1e43f697fbe/packages/react-app-polyfill/package.json#L24

Reply:
By pinning I'm afraid that we provoke dependencies duplication in the dependency tree. By being more loose we allow CRA patch upgrade without the need to upgrade this preset again.

@mrmckeb I'm waiting for your thought on that before making the change

Reply:
Hi @Spy-Seth, let's see how this goes then. We can always lock it down later.